### PR TITLE
use ADAL generic value

### PR DESCRIPTION
**Pull request by jennyf19**
we need to use the adal generic so devices using adal and msal will not be impacted. but just for writing, lookup should be just service and account.

### CODE REVIEW

**Comment by jmprieur** [comment]
is it a breaking change? for MSAL only apps?

**Comment by jennyf19** [comment]
@jmprieur as discussed off-line, not a breaking change as the look up has not changed.

File: src/client/Microsoft.Identity.Client/Platforms/iOS/iOSBrokerConstants.cs [hunk 1]
```diff
@@ -36,7 +36,7 @@ internal class iOSBrokerConstants
         public const string BrokerKeyLabel = "Broker Key Label";
         public const string BrokerKeyComment = "Broker Key Comment";
         public const string BrokerKeyStorageDescription = "Storage for broker key";
-        public const string LocalSettingsContainerName = "MicrosoftIdentityClient";
+        public const string LocalSettingsContainerName = "ActiveDirectoryAuthenticationLibrary";
```
Comment (bgavrilMS) [inline src/client/Microsoft.Identity.Client/Platforms/iOS/iOSBrokerConstants.cs]: Same questions as @jmprieur asked:

- what is the impact of this change to existing apps? is it a breaking change? will it result in users having to login again?

Comment (jennyf19) [reply src/client/Microsoft.Identity.Client/Platforms/iOS/iOSBrokerConstants.cs]: i sent you an email with the details.

**Comment by jennyf19** [comment]
closing...not doing this as discussed.

**Pull request closed without merging**